rpc_sidecar/README.md

```diff
@@ -21,7 +21,13 @@ The sidecar maintains a TCP connection with the node and communicates using a cu
     - request to submit a transaction for execution
     - request to speculatively execute a transaction 
 
-The node does not interpret the data it sends where it's not necessary. For example, most database items are sent as opaque byte arrays and the sidecar is responsible for interpreting them. This leaves the sidecar in control of the data it receives and allows it to be more flexible in how it handles it.
```

Comment:
Is this not applicable anymore?

Reply:
Yeah, actually some of the validation was moved to node, because this decoupling led to a very sub-optimal and talkative back-and-forth between sidecar and the node. So it surely is not true for all the endpoints.